libraries/rush-lib/src/logic/PackageJsonUpdater.ts

```diff
@@ -342,10 +343,12 @@ export class PackageJsonUpdater {
     dependencyAnalyzer: DependencyAnalyzer,
     options: IPackageJsonUpdaterRushAddOptions
   ): Promise<IUpdateProjectOptions[]> {
-    const { packagesToUpdate, devDependency, peerDependency, updateOtherPackages } = options;
+    const { projects, packagesToUpdate, devDependency, peerDependency, updateOtherPackages } = options;
```

Comment:
Would it be more intuitive to design the API as `_updateProjects(projects, dependencyAnalyzer, { subspace, ... })`?

Reply:
Never mind. It's a private function.